### PR TITLE
[7.0] Greenify branch (codestyle + test-fix)

### PR DESCRIPTION
**Pull request by katyukha**

### CODE REVIEW

**Comment by dreispt** [comment]
Not green yet, but no more style problems.
Kudos! :+1:

File: project_sla/test/project_sla.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@
   There should be Service Level info generated on the Issue
 -
   !assert {model: project.issue, id: issue1, string: Issue should have calculated service levels}:
-    - len(sla_control_ids) == 2
+    - len(sla_control_ids) == 1
```
Comment (pedrobaeza) [inline project_sla/test/project_sla.yml:38]: Why this change in the test?

**Comment by pedrobaeza** [comment]
The rest LGTM

File: project_sla/test/project_sla.yml (hunk 1 above)
Comment (katyukha) [inline project_sla/test/project_sla.yml:38]: Look at [8.0 version of tests](https://github.com/OCA/project-service/blob/8.0/project_sla/test/project_sla.yml#L39)

Comment (pedrobaeza) [inline project_sla/test/project_sla.yml:38]: Right

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:
